src/foam/nanos/client/ClientBuilder.js

```diff
@@ -83,15 +94,34 @@ foam.CLASS({
             ]
           };
 
-          var references = [];
+          // var references = [];
+
+          let nspec = foam.nanos.boot.NSpec;
+          let query = self.EQ(self.NSpec.SERVE, true);
+
+          if ( ! self.authenticate ) {
+            query = self.AND(query, self.EQ(self.NSpec.AUTHENTICATE, false));
```

Comment:
Where is this used?

Reply:
its not yet, its still a todo on the service, but ZAC uses this property